galleon-pack/wildfly-feature-pack-build.xml

```diff
@@ -87,6 +87,8 @@
                 <extension>org.wildfly.extension.microprofile.fault-tolerance-smallrye</extension>
                 <extension>org.wildfly.extension.microprofile.jwt-smallrye</extension>
                 <extension>org.wildfly.extension.microprofile.openapi-smallrye</extension>
+                <extension>org.wildfly.extension.microprofile.reactive-messaging-smallrye</extension>
+                <extension>org.wildfly.extension.microprofile.reactive-streams-operators-smallrye</extension>
```

Comment:
1) Is this not supported in a domain server?

2) This same thing is needed in ee-9/feature-pack/wildfly-feature-pack-build.xml

Reply:
Weird, I added them to domain this week to fix some problems @jstourac was seeing. Will re-add them